src/client/pages/Toerisme/Toerisme.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Page,
+  PageContent,
+  PageHeading,
+  Panel,
+  Heading,
+} from '../../components';
+import classnames from 'classnames';
+import styles from './Toerisme.module.scss';
+import { AppRoutes, ChapterTitles } from '../../../universal/config/index';
+import { ChapterIcon, Linkd } from '../../components';
+
+import { useAppStateGetter } from '../../hooks/useAppState';
+
+export default function Toerisme() {
+  const { TOERISME } = useAppStateGetter();
+  // Wait for design to show multiple instances for now we take the first in array
+  const info = TOERISME?.content;
+  return (
+    <Page className={styles.Toerisme}>
+      <PageHeading
+        backLink={{
+          to: AppRoutes.HOME,
+          title: 'Home',
+        }}
+        icon={<ChapterIcon />}
+      >
+        {ChapterTitles.TOERISME}
+      </PageHeading>
+      <PageContent>
+        <p>
+          Hieronder vind u een overzicht van uw aanvragen voor toeristische
+          verhuur
+        </p>
+        <p>
+          <Linkd
+            external={true}
+            href="https://www.amsterdam.nl/wonen-leefomgeving/wonen/vakantieverhuur/"
+          >
+            Meer informatie over regels voor Particuliere vakantieverhuur
+          </Linkd>
+          <br />
+          <Linkd
+            external={true}
+            href="https://www.amsterdam.nl/veelgevraagd/?productid=%7BF5FE8785-9B65-443F-9AA7-FD814372C7C2%7D"
+          >
+            Meer over toeristenbelasting
+          </Linkd>
+        </p>
+        {info?.map((infoItem) => (
```

Comment:
De opmaak/style van deze items wordt al gebruikt in de applicatie. In dit geval was dit `<InfoItem />`. Dit heb ik gebruikt in de code die in de branch is geland. Het had wel wat refactoring nodig, echter het scheelt weer wat onderhoud aan individuele styling.

Als je iets tegenkomt dat gegeneraliseerd kan worden kun je daar beste een component van maken / bestaand iets aanpassen zodat het ook aan deze use-case voldoet.